metrics-facade-base/src/main/java/com/ringcentral/platform/metrics/producers/SystemMetricsProducer.java

```diff
@@ -16,7 +18,11 @@ public class SystemMetricsProducer implements MetricsProducer {
     private final ClassesMetricsProducer classesMetricsProducer;
 
     public SystemMetricsProducer() {
-        this(null);
+        this(null, false);
+    }
+
+    public SystemMetricsProducer(boolean isDimensional) {
```

Comment:
This is a minor thing and is actually a matter of taste, but the prefix 'is' for boolean fields/params seems redundant.
https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter

Reply:
fixed, thank you.